server/views/icons/actions/arrow-big.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 20 26"><path class="icon-sprite__shape" fill="#1d1d1d" d="M18.71 15.29a1 1 0 0 0-1.41 0l-6.3 6.3V2a1 1 0 0 0-2 0v19.59l-6.29-6.3A1 1 0 0 0 1.3 16.7l8 8a1 1 0 0 0 1.41 0l8-8a1 1 0 0 0 0-1.41z"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 20 26"><path class="icon__shape" fill="#1d1d1d" d="M18.71 15.29a1 1 0 0 0-1.41 0l-6.3 6.3V2a1 1 0 0 0-2 0v19.59l-6.29-6.3A1 1 0 0 0 1.3 16.7l8 8a1 1 0 0 0 1.41 0l8-8a1 1 0 0 0 0-1.41z"/></svg>
```

Comment:
Interesting - since this is so tied to our implementation - not the SVG itself - should we add this to the icon component? I'd say the optimiser should strip these.